packages/tlinks/src/ui/FarcasterContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import { NeynarAuthButton, SIWN_variant, useNeynarContext } from "@neynar/react";
+import { useEffect, useState } from "react";
+import { FarcasterFrame } from "./FarcasterFrame.tsx";
+import { Buffer } from 'buffer';
+import type { FarcasterContainerProps, FarcasterUser } from "../utils/constants.ts";
+globalThis.Buffer = Buffer;
+
+
+export const FarcasterContainer = ({
+    chain,
+    scriptURI,
+}: FarcasterContainerProps) => {
+
+    const { user } = useNeynarContext();
+    const [farcaster, setFarcaster] = useState<FarcasterUser | null>(null);
+
+    useEffect(() => {
+        const setupFarcasterUser = async () => {
+            if (user) {
+                const address = await chrome.runtime.sendMessage({ type: "getConnectedAccount" });
```

Comment:
tlink package can run in both chrome extension and on website.
please remove the use of `chrome` api here.

![CleanShot 2024-11-15 at 10 16 03@2x](https://github.com/user-attachments/assets/a71ce989-7df7-4d04-8ccd-9061361d439b)